### PR TITLE
Rel item num

### PR DESCRIPTION
**Pull request by davidmdm**

### CODE REVIEW

**Comment by e2-cloud** [comment]
:star: PR built and available in a preview environment **lvlstudio-e2-index-pr-3** [here](http://e2-index.jx-lvlstudio-e2-index-pr-3.e2-dev.com)

**Comment by e2-cloud** [comment]
:star: PR built and available in a preview environment **lvlstudio-e2-index-pr-3** [here](http://e2-index.jx-lvlstudio-e2-index-pr-3.e2-dev.com)

File: scripts/migrate.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+const conn = knex({
+  client: 'pg',
+  version: '11.5',
+  connection: connectionUrl,
+  debug: false,
+  useNullAsDefault: true,
+});
+
+const schemas = ['860907', '860907-test', '111111'];
```
Comment (mlaflamm) [inline scripts/migrate.js]: ?????????

**Comment by e2-cloud** [comment]
:star: PR built and available in a preview environment **lvlstudio-e2-index-pr-3** [here](http://e2-index.jx-lvlstudio-e2-index-pr-3.e2-dev.com)

**Comment by e2-cloud** [comment]
:star: PR built and available in a preview environment **lvlstudio-e2-index-pr-3** [here](http://e2-index.jx-lvlstudio-e2-index-pr-3.e2-dev.com)

**Pull request merged**